.run/Template Mocha.run.xml

```diff
@@ -0,0 +1,14 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="true" type="mocha-javascript-test-runner">
+    <node-interpreter>project</node-interpreter>
+    <node-options />
+    <working-directory />
+    <pass-parent-env>true</pass-parent-env>
+    <ui>qunit</ui>
+    <extra-mocha-options />
+    <test-kind>DIRECTORY</test-kind>
+    <test-directory />
+    <recursive>false</recursive>
+    <method v="2" />
+  </configuration>
+</component>
```

Comment:
what's this file?

Reply:
Ooops, never mind I will remove it. It's a shared run configuration for WebStorm.